Watchman.Discord/Areas/Initialization/Services/ResponsesInitService.cs

```diff
@@ -39,7 +44,7 @@ private async Task AddNewResponses(IReadOnlyCollection<Response> responsesToAdd)
                 return;
             }
 
-            var command = new AddResponsesCommand(responsesToAdd);
+            var command = new UpdateResponsesCommand(responsesToAdd);
```

Comment:
i to działa? bo z tego co ja widzę to
```csharp
 var responsesToAdd = defaultResponses
     .Where(def => responsesInBase.All(@base => @base.OnEvent != def.OnEvent))
     .ToList();
```
oznacza że do responsesToAdd są przypisywane odpowiedzi z pliku resources, czyli każda odpowiedź ma nowe ID
a w handlerze UpdateResponsesCommand używasz AddOrUpdate które porównuje po Entity.ID

Reply:
oj faktycznie, rozpędziłem się za bardzo. Już tam naprawiam